### ISSUE TITLE
[insights] alt as nothing needs additional clean up

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#1059]
Some empty images don't have a value of nothing appropriately. the `el.getAttribute('alt')` when nothing must be returning `null` when empty or something; need to dig on specific content to know for sure.
- [x] make a test page that has all the different cases to ensure each one does it
- [x] resolve difference between `alt` and `alt=""` and there being no alt as all 3 are coming up "null" in the current interpretation on the backend (leading to front end error statement being false at times)
